pilot/pkg/features/pilot.go

```diff
@@ -104,13 +105,13 @@ var (
 			" EDS pushes may be delayed, but there will be fewer pushes. By default this is enabled",
 	).Get()
 
-	SendUnhealthyEndpoints = env.RegisterBoolVar(
+	SendUnhealthyEndpoints = atomic.NewBool(env.RegisterBoolVar(
```

Comment:
I do wonder if we should change the `env` package -- possibly at the same time as making it use generics -- to make it so everything is atomic?

This problem is universal, just happens that one test triggers it most often

Reply:
> I do wonder if we should change the env package -- possibly at the same time as making it use generics -- to make it so everything is atomic?

Yes.
Should I change others to also this model for now and update env package later when we have generics?